docs/release-notes.md

```diff
@@ -9,20 +9,57 @@ description: >
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.8.0`. For
+The latest tagged release is `2.8.1`. For
 installation instructions see our
 [Install](/docs/reference/cluster-operator/install) page.
 
-The latest CLI release is `2.8.0`, available from
+The latest CLI release is `2.8.1`, available from
 [GitHub](https://github.com/storageos/go-cli/releases).
 
 # Upgrading
 
 To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
-## 2.8.0 - Release 2022-06-29
+## 2.8.1 - Release 2022-08-02
+
+### New
+
+k8s
+
+* Install resource quota for GKE
+* Disable scheduler option in storageoscluster CR
+* Operator:
+  * Add snapshot controller and related CRDs if not present
+  * Pod Disruption Budget support for k8s v1.25
+* Improve logging on kubectl plugin <https://github.com/storageos/kubectl-storageos/pull/194>
+
+Control Plane
+
+* Adds randomisation behaviour of placement
```

Comment:
I would recommend removing `* Adds randomisation behaviour of placement` it's a bit disingenuous 

These release notes should be sufficient for the control plane
```
 - Reduced the of amount of crash loop backoffs when installing via the helm chart
 - Reduced the impact of ListVolumes on etcd (significantly, in the case of clusters with lots of volumes) 
 - Fixed an issue where formatting would timeout due to large TRIM writes being sent across the network 
 - Fixed an issue where volume deployments would all be scheduled on the same nodes when deploying multiple PVC at the same time 
 - Automatically round up storage requests to align with blocksize, instead of rejecting requests 
 ```

Reply:
fixed